### ISSUE TITLE
2.1 error when making changes

### ISSUE THREAD

**Issue by yatil** [jekyll/jekyll-watch#75]
After updating to version 2.1.0 ('bundle update'), changing any file resulted in an error as you can see below. Downgrading to 2.0.0 works for me.

```bash
❯ bundle exec jekyll serve
Configuration file: /Users/yatil/Library/Mobile Documents/com~apple~CloudDocs/projects/wai-about-wai/_config.yml
            Source: /Users/yatil/Library/Mobile Documents/com~apple~CloudDocs/projects/wai-about-wai
       Destination: /Users/yatil/Library/Mobile Documents/com~apple~CloudDocs/projects/wai-about-wai/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
      Remote Theme: Using theme w3c/wai-website-theme
                    done in 3.132 seconds.
 Auto-regeneration: enabled for '/Users/yatil/Library/Mobile Documents/com~apple~CloudDocs/projects/wai-about-wai'
    Server address: http://127.0.0.1:4000/wai-about-wai/
  Server running... press ctrl-c to stop.
      Regenerating: 1 file(s) changed at 2018-10-10 12:37:21
E, [2018-10-10T12:37:21.434728 #51330] ERROR -- : exception while processing events: undefined local variable or method `source_path' for Jekyll::Watcher:Module Backtrace:
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/jekyll-watch-2.1.0/lib/jekyll/watcher.rb:63:in `block (2 levels) in listen_handler'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/jekyll-watch-2.1.0/lib/jekyll/watcher.rb:63:in `each'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/jekyll-watch-2.1.0/lib/jekyll/watcher.rb:63:in `block in listen_handler'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/listen-3.1.5/lib/listen/event/config.rb:23:in `call'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/listen-3.1.5/lib/listen/event/processor.rb:115:in `_process_changes'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/listen-3.1.5/lib/listen/event/processor.rb:19:in `block in loop_for'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/listen-3.1.5/lib/listen/event/processor.rb:15:in `loop'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/listen-3.1.5/lib/listen/event/processor.rb:15:in `loop_for'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/listen-3.1.5/lib/listen/event/loop.rb:84:in `_wait_for_changes'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/listen-3.1.5/lib/listen/event/loop.rb:42:in `block in setup'
 -- /Users/yatil/.rvm/gems/ruby-2.4.1/gems/listen-3.1.5/lib/listen/internals/thread_pool.rb:6:in `block in add'
```

**Comment by ashmaroli** [comment]
@yatil This issue will be resolved by #73

**Comment by yatil** [comment]
@ashmaroli Thanks for the heads-up (I should have looked in the PRs as well…)
